### DISCUSSION TITLE
add ascii display

### DISCUSSION THREAD

**Discussion by carson-coder** [SebLague/Digital-Logic-Sim#75] [Ideas]
i want to display text

**Comment by jerrecode** [comment]
maybe a square matrix display would be cool to accomplish this

**Reply by carson-coder** [reply]
I could work on this later maybe

**Reply by carson-coder** [reply]
I made a pull request that added a letter display already tho. It's one of the closed ones

**Reply by jerrecode** [reply]
Oh that's very sad would have been cool.

but why was it closed would it be to complicated to put in?

or maybe it is overshadowed by requests with more priority?

**Comment by parshwa282011** [comment]
> i want to display text

You could do some really complicated stuff with the dot display.
